apps/hash-graph/bin/cli/src/main.rs

```diff
@@ -5,27 +5,33 @@ mod args;
 mod error;
 mod subcommand;
 
+use std::sync::Arc;
+
 use error_stack::Result;
 use graph::load_env;
 
-use self::{args::Args, error::GraphError, subcommand::Subcommand};
+use self::{args::Args, error::GraphError};
 
-#[tokio::main]
```

Comment:
With this change it's possible to remove the `macros` feature from `tokio` I think.

Reply:
let's try it! removed in [c413faf](https://github.com/hashintel/hash/pull/3091/commits/c413faf59488886df56f0a4abc5162c3cf3412b3)